zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/PostImporterQueueFromIncidentHandler.java

```diff
@@ -69,7 +70,7 @@ public void updateEntity(
         .setIntent(intent)
         .setKey(record.getKey())
         .setPosition(record.getPosition())
-        .setCreationTime(OffsetDateTime.now())
+        .setCreationTime(OffsetDateTime.now().truncatedTo(ChronoUnit.MILLIS))
```

Comment:
❓ Not sure whether we can simply do that? What was the reason behind this?

Reply:
Elasticsearch actually trancates the time, there you send the expectedEntity which is what is flushed to ELS, you see the document on ELS, and then you see the result of the get in responseEntity (which shows the document in ELS it is as we expect) 
![image](https://github.com/user-attachments/assets/13464398-6b97-44c8-b0af-b5768c0777e4)
![image](https://github.com/user-attachments/assets/bb468cff-51c3-4654-8b64-3c0080c9bae2)

Reply:
But isn't this something we have to do everywhere then? 🤔